docs/index.md

```diff
@@ -1,4 +1,128 @@
-<img style="float: left; padding-top: 7px; padding-right: 10px; padding-bottom: 20px;" src="images/atomist-logo-source.jpg">
+<img style="float:left; padding-top:7px; padding-right:10px; padding-bottom:15px;" src="images/atomist-logo.png">
+
+# Atomist Overview
+
+Atomist enables you to ship software in less time and respond to
+operational events more quickly.  By automating tasks, integrating
+tools, and putting timely, correlated, and actionable information in
+front of your team, Atomist makes software development and operational
+response faster, more standardized, and more fun.
+
+Timely, correlated, and actionable information results from capturing
+your processes and operations in code.  Atomist knows the platforms
+and tools you already use: from GitHub to BitBucket, from Trello to
+Jira, from Travis CI to Jenkins, from Kubernetes to Spinnaker.
+Atomist understands your code: Java, C#, JavaScript, Scala, Python,
+Clojure, even Dockerfiles and POMs.  Most importantly, Atomist
+understands the relationship between your code, your tools, your
+environments, and your running services and brings this information to
+your team members where they live: chat.
```

Comment:
"... brings this information to ~your team members~ *you* where you live: chat."